packages/create-svelte/types/index.d.ts

```diff
@@ -0,0 +1,9 @@
+import { Options } from './internal';
+
+/**
+ * Create a new SvelteKit project.
+ *
```

Comment:
I think we mostly don't have a blank line in between the jsdoc description and parameters elsewhere
```suggestion
```